container/features/src/main/resources/features.xml

```diff
@@ -1416,7 +1416,7 @@
     </feature>
     <feature name="opennms-jest" version="${project.version}" description="OpenNMS :: Features :: Jest :: Feature definition">
         <details>Feature definition for Jest (ElasticSearch Java ReST Client)</details>
-        <bundle>wrap:mvn:com.google.code.gson/gson/${jestGsonVersion}</bundle>
+        <bundle>mvn:com.google.code.gson/gson/${jestGsonVersion}</bundle>
```

Comment:
Was encountering class-loading issues when using gson, so I bumped it to a newer version that includes the OSGi meta-data to make sure that wasn't the problem.